src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+    <link href="https: //fonts.googleapis.com/css2? family= Roboto:wght@500 & display=swap" rel="stylesheet">
+  </head>
+  <body class="card">
+    <selection class="card__item" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        class="card__imac" 
```

Comment:
```suggestion
        class="card__image" 
```
There could be different images not only imac so card__image is better className

Reply:
done